source/_includes/asides/component_navigation.html

```diff
@@ -37,6 +37,16 @@
     {%- endif -%}.
   </div>
 
+  {% if page.works_with %}
+    {%- for type in page.works_with -%}
```

Comment:
All custom variables are prefixed `ha_` could you please adjust it for this one to @zsarnett? Thanks 👍